services/s3/src/main/resources/codegen-resources/customization.config

```diff
@@ -6,6 +6,17 @@
     "Error": "S3Error",
     "Object": "S3Object"
   },
+  "customSdkShapes": {
+    "shapes":{
+      "PartType":{
```

Comment:
Should we rename it to `sdkPartType` so that it doesn't collide with future service shapes?

Reply:
renamed to SdkPartType